### PR TITLE
Search/filter pills

### PR DESCRIPTION
**Pull request by PaulSchroeder**

### CODE REVIEW

**Comment by danieleguido** [comment]
Hi @PaulSchroeder can you fix the pending conflicts?

**Comment by PaulSchroeder** [comment]
@danieleguido fixed conflict :)

File: src/components/modules/FilterFacetYear.vue [hunk 1]
```diff
@@ -1,12 +1,16 @@
 <template lang="html">
   <div v-bind:class="{'mb-3': expanded}">
-    <base-title-bar>
+    <div
+      class="border-top border-tertiary pt-1px" style="margin:auto -1em">
+      <div class="border-top" />
```
[the remark this answers is not in the document]
Comment (PaulSchroeder) [reply src/components/modules/FilterFacetYear.vue]: the first part is for the double border, 
the second part is a hack to break the margin set by the container.

`border-style: double` is not the same, both lines would have the same color

File: src/components/SearchFilters.vue [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <template lang="html">
   <div id="search-filters" class="px-3">
-    <div v-for="(filter, index) in filters">
+    <div v-bind:class="'filter-wrapper filter-'+filter.type" v-for="(filter, index) in filters">
```
[the remark this answers is not in the document]
Comment (PaulSchroeder) [reply src/components/SearchFilters.vue]: good call, changed the syntax as you suggest.

**Pull request merged**